Xamarin.PropertyEditing.Mac/Controls/Variations/CreateVariantView.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class CreateVariantView
+		: NSView
+	{
+		private const int ControlSpacing = 7;
+		private const int HorizontalControlSpacing = 10;
+		internal const int RightEdgeMargin = 12;
+
+		private IHostResourceProvider hostResources;
+
+		public CreateVariantView (IHostResourceProvider hostResources, CGSize windowWidthHeight, CreateVariantViewModel createVariantViewModel)
+		{
```

Comment:
`ArgumentNullException` check for `createVariantViewModel`

Reply:
Yes, missed it.